core/src/commonMain/kotlin/com/xebia/functional/xef/auto/AIRuntime.kt

```diff
@@ -1,31 +1,55 @@
 package com.xebia.functional.xef.auto
 
-import com.xebia.functional.xef.embeddings.OpenAIEmbeddings
-import com.xebia.functional.xef.env.OpenAIConfig
-import com.xebia.functional.xef.llm.openai.KtorOpenAIClient
-import com.xebia.functional.xef.llm.openai.LLMModel
+import com.aallam.openai.api.logging.LogLevel
+import com.aallam.openai.api.logging.Logger
+import com.aallam.openai.client.LoggingConfig
+import com.aallam.openai.client.OpenAI
+import com.aallam.openai.client.OpenAIConfig
+import com.xebia.functional.xef.embeddings.Embeddings
+import com.xebia.functional.xef.env.getenv
+import com.xebia.functional.xef.llm.AIClient
+import com.xebia.functional.xef.llm.LLMModel
+import com.xebia.functional.xef.llm.openai.OpenAIClient
+import com.xebia.functional.xef.llm.openai.OpenAIEmbeddings
 import com.xebia.functional.xef.vectorstores.LocalVectorStore
 import kotlin.jvm.JvmStatic
 import kotlin.time.ExperimentalTime
 
-data class AIRuntime<A>(val runtime: suspend (block: AI<A>) -> A) {
+data class AIRuntime<A, C : AIClient>(
```

Comment:
```suggestion
data class AIRuntime<A>(
```

This generic parameter is unused, it's always used as `*`.